src/view-components/widget/widget.component.tsx

```diff
@@ -2,29 +2,49 @@ import React, { FunctionComponent } from "react";
 import { coreWidgets } from "../core-views";
 
 export default function Widget(props) {
-  const [widget, setWidget] = React.useState();
+  const [widget, setWidget] = React.useState<WidgetConfigType | null>(null);
```

Comment:
I would use `const [widget, setWidget] = React.useState<WidgetConfigType >(null);` Initialising it with null is sufficient.

Reply:
Done